Project.toml

```diff
@@ -12,7 +12,11 @@ SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-Polynomials = ">= 0.1.0"
+Polynomials = "0.6.0"
+FFTW = "1.0"
```

Comment:
This won't actually restrict AbstractFFTs, since FFTW permits any AbstractFFTs >= 0.3.